examples/deck.gl/i3s/app.js

```diff
@@ -41,47 +31,23 @@ const INITIAL_VIEW_STATE = {
   maxZoom: 30,
   zoom: 14.5
 };
-/* */
-//NY
-/*
-const INITIAL_VIEW_STATE = {
- longitude: -73.97785639190995,
- latitude: 40.75262236078426,
- height: 600,
- width: 800,
- pitch: 45,
- maxPitch: 60,
- bearing: 0,
- minZoom: 2,
- maxZoom: 30,
- zoom: 14.5
-};
-*/
 
-//philadelphia_Bldgs_text
-/*
-const INITIAL_VIEW_STATE = {
-  longitude: -75.16725679895995,
-  latitude: 39.95667467886362,
-  height: 600,
-  width: 800,
-  pitch: 45,
-  maxPitch: 60,
-  bearing: 0,
-  minZoom: 2,
-  maxZoom: 30,
-  zoom: 14.5
+const STATS_WIDGET_STYLE = {
```

Comment:
Nit: move good defaults into probe.gl? We should not need 10 lines of config here just to instantiate a widget - it adds a lot of clutter to an example that we want to be as simple as possible. 

I think I have made this comment many times I feel maybe this is the time it happens? :)